src/ripple/protocol/impl/STVar.cpp

```diff
@@ -49,7 +49,7 @@ STVar::STVar (STVar const& other)
         p_ = other.p_->copy(max_size, &d_);
 }
 
-STVar::STVar (STVar&& other)
+STVar::STVar (STVar&& other) noexcept
 {
     if (other.on_heap())
```

Comment:
`other.on_heap()` is not noexcept.
`p_ = other.p_->move(max_size, &d_)` is not noexcept.